recipes/itmlogic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "itmlogic" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fa65eb9bff2a0ff3798bc10d6962874aa594d4ed9826d4a36c23cb3b81115e51
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - setuptools_scm
```

Comment:
I'm pretty sure you don't need this on run. Can you try removing it in the feedstock later?

Reply:
You do need this on run. I added this in commit `394334f` which solved previous failures without it.

Reply:
Interesting. I use setuptools_scm with the "write to file" option, so I never needed it on `run`.

Reply:
Even without `write_to_file` you shouldn't need it. I briefly look at the source of `itmlogic` and cannot see the need. Feel free to ping me on the feedstock with a PR where the build fails and I can have a look.

`write_to_file` should write the version to an additional file but the version is already persisted in the package metadata in the case of an `sdist`. Additionally in an sdist, the VCS history is not available anymore, so `setuptools_scm` actually cannot do its magic anymore.